### PR TITLE
Add repo for image promotion

### PR DESCRIPTION
**Pull request by ameukam**
Add a sub-repo to investigate promotion to Artifact Registry.

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Comment by ameukam** [comment]
FYI @upodroid @puero

**Comment by upodroid** [comment]
Thanks.

I'm also testing this on my own infra for rapid iteration.

https://github.com/kubernetes-sigs/promo-tools/pull/609
https://github.com/upodroid/k8s-ar-testing/tree/master/k8s.gcr.io

**Review by upodroid — APPROVED** [review]
LGTM

File: registry.k8s.io/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```
Comment (BenTheElder) [inline registry.k8s.io/OWNERS:1]: why is this in this repo instead of k8s.io?

Comment (ameukam) [reply registry.k8s.io/OWNERS:1]: To test image promotion to Artifact registry. Trying to replicate process without block it with experimentation.

**Pull request closed without merging**